licence-data.php

```diff
@@ -1,18 +0,0 @@
-<?php
```

Comment:
This file should not be removed

Reply:
Hmm, this is going to be a continuous problem with our commits. It gets deleted when live testing.

Added back in https://github.com/wp-media/wp-rocket/pull/2470/commits/338decfad8bc2418078c750b2eace051f9da9676